source/Nevermore/Mapping/GuidPrimaryKeyHandler.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Nevermore.Mapping
+{
+    class GuidPrimaryKeyHandler : PrimitivePrimaryKeyHandler<Guid>
+    {
+        public override object FormatKey(string tableName, int key)
```

Comment:
This feels wrong and only works because we actually don't call this method. If we did this cast would fail https://github.com/OctopusDeploy/Nevermore/blob/e8f71a42696af4d7cd26a65732534f900bd96cd3/source/Nevermore/Advanced/WriteTransaction.cs#L232